getting_started/step_by_step/your_first_game.rst

```diff
@@ -206,6 +234,36 @@ or ``false`` if it isn't.
         else:
             $AnimatedSprite.stop()
 
+ .. code-tab:: csharp
+
+    public override void _Process(float delta)
+    {
+        var velocity = new Vector2();
+        if(Input.IsActionPressed("ui_right")) {
+            velocity.x += 1;
+        }
+
+        if(Input.IsActionPressed("ui_left")) {
+            velocity.x -= 1;
+        }
+        
+        if(Input.IsActionPressed("ui_down")) {
+            velocity.y += 1;
+        }
+
+        if(Input.IsActionPressed("ui_up")) {
+            velocity.y -= 1;
+        }
+
+        var animated_sprite = GetNode("AnimatedSprite") as AnimatedSprite;
```

Comment:
Maybe we should use PascalCase here? And maybe we could also consider changing the safe cast to the direct cast, since we depend on an assumption that it wouldn't be null and has the right type in this case.